core/__tests__/models/app/app.ts

```diff
@@ -350,6 +364,53 @@ describe("models/app", () => {
       expect(testCounter).toBe(1);
     });
 
+    describe("default option values", () => {
```

Comment:
I have no idea if this is the right file to do this in- it seems like at the moment there isn't a test for modules/optionHelper

Reply:
Moved to models/options